lexicons/com/atproto/admin/getSubjectState.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.getSubjectState",
```

Comment:
Do you envision having a separate endpoint for getting the subject state specific to a particular service?  Or beyond that, fetching the subject state for multiple services at once?

Reply:
yeah - i view this as just being "what is this particular service's state for this subject?" then the moderation service could have some higher-level end points for collating state from across services